ddtrace/ext/ci.py

```diff
@@ -442,6 +510,25 @@ def extract_bitrise(env):
     }
 
 
+def extract_buddy(env):
```

Comment:
is it possible to add this in a separate PR?

since you have two changes listed under release notes, it probably means these changes could be separated?